lib/plugins/max-scroll-tracker.js

```diff
@@ -202,8 +202,9 @@ class MaxScrollTracker {
    * @return {number}
    */
   getPagePath() {
-    return parseUrl(
-        this.tracker.get('page') || this.tracker.get('location')).path;
+    const url = parseUrl(
```

Comment:
Does this change (L205-207) belong in this PR? The rest are about the clean URL plugin, while this is the max-scroll-tracker